checkmate/models/data/custom_rule.py

```diff
@@ -0,0 +1,36 @@
+"""Model for our own blocking rules."""
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import ARRAY
+
+from checkmate.checker.url.reason import Reason
+from checkmate.db import BASE
+
+
+class CustomRule(BASE):
+    """Rule about blocking a particular resource."""
+
+    __tablename__ = "custom_rule"
+    __table_args__ = (sa.Index("ix__hash", "hash"),)
```

Comment:
Possibly, but I've gone through a few different variations with different settings that aren't the default while I was experimenting, which wouldn't be supported. It turns out B-Tree + C collations is the best.

* The collation is done on the column anyway
* B-Tree is the default, so this could go back now

I tried `HASH` which was horribly confusing, and a few other things, but this was the fastest by a mile.

Reply:
I've updated this as suggested and added a note linking to the docs